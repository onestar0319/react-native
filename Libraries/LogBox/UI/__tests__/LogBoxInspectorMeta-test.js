/**
 * Copyright (c) Facebook, Inc. and its affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * @format
 * @emails oncall+react_native
 * @flow
 */

'use strict';

const React = require('react');
const LogBoxInspectorMeta = require('../LogBoxInspectorMeta').default;
const render = require('../../../../jest/renderer');

describe('LogBoxInspectorMeta', () => {
  it('should render meta information', () => {
    const output = render.shallowRender(<LogBoxInspectorMeta />);

    expect(output).toMatchSnapshot();
  });
});
