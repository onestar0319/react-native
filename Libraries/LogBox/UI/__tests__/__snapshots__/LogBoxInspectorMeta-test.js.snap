// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LogBoxInspectorMeta should render meta information 1`] = `
<View
  style={
    Object {
      "marginTop": 15,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "marginBottom": 10,
        "paddingHorizontal": 12,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "rgba(255, 255, 255, 1)",
          "flex": 1,
          "fontSize": 20,
          "fontWeight": "600",
          "includeFontPadding": false,
          "lineHeight": 20,
        }
      }
    >
      Meta
    </Text>
  </View>
  <View
    style={
      Object {
        "flexDirection": "row",
        "justifyContent": "space-between",
        "paddingBottom": 100,
        "paddingLeft": 25,
        "paddingRight": 25,
      }
    }
  >
    <View
      style={
        Object {
          "flex": 0,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "rgba(255, 255, 255, 1)",
            "flex": 0,
            "flexGrow": 0,
            "fontSize": 14,
            "includeFontPadding": false,
            "lineHeight": 20,
          }
        }
      >
        Engine
      </Text>
    </View>
    <View
      style={
        Object {
          "flex": 0,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "rgba(255, 255, 255, 1)",
            "flex": 0,
            "flexGrow": 0,
            "fontSize": 14,
            "includeFontPadding": false,
            "lineHeight": 20,
          }
        }
      >
        Unknown
      </Text>
    </View>
  </View>
</View>
`;
